test/extensions/transport_sockets/tls/ocsp/ocsp_test.cc

```diff
@@ -189,16 +189,28 @@ TEST_F(Asn1OcspUtilityTest, ParseMethodWrongTagTest) {
   expectThrowOnWrongTag(Asn1OcspUtility::parseResponseStatus);
 }
 
-TEST_F(Asn1OcspUtilityTest, ParseResponseDataBadResponderIdVariantTest) {
+TEST_F(Asn1OcspUtilityTest, ParseResponseDataUnsupportedVersionTest) {
   std::vector<uint8_t> data = {
       // SEQUENCE
-      0x30,
-      7,
-      // version
+      0x30, 5,
+      // invalid version field, the value of v2 is 1
```

Comment:
There's no such thing as v2. Better to say "Invalid version field. The value of v1 is 0, not 1."

Reply:
Done.